src/lib/components/chips/Chips.component.js

```diff
@@ -0,0 +1,114 @@
+//@flow
+import * as defaultTheme from "../../style/theme";
+import styled, { css } from "styled-components";
+import React from "react";
+import { lighten } from "polished";
+import { mergeTheme } from "../../utils";
+
+type Props = {
+  text: string,
+  color?: String,
+  variant?: Variant,
+  icon?: Node,
+  buttonIcon?: Node,
+  onClick?: any => void,
```

Comment:
It would be nicer to have a remove button as part of this component as you do in the storybook. 
We can add ‘onRemove?: any => void’ in the Props. Then if ‘onRemove’ is defined we display the remove Button, otherwise not.